messages/de-spdth/motion.php

```diff
@@ -0,0 +1,55 @@
+<li><strong>Unterstützung finden:</strong> Nach der Bestätigung erhältst du einen Link, den du Interessierten weiterleiten kannst. Über diesen Link kann dein Antrag gelesen und von Parteimitgliedern unterstützt werden, die damit mit Dir gemeinsam den Antrag stellen. Bitte beachte, dass sich auch die Unterstützer*innen deines Antrags mit ihren Zugangsdaten für das Grüne Netz einloggen und darüber verifizieren müssen.</li>
+<li><strong>Antrag einreichen:</strong> Sobald sich die notwendigen %MIN% Unterstützer*innen gefunden haben, musst du den angelegten Antrag noch offiziell bei der Antragskommission einreichen. Dies veranlasst Du auch über das SPD-Antragstool. Wir informieren dich selbstverständlich per E-Mail, sobald die notwendige Zahl der Unterstützer*innen erreicht ist.</li>
+</ol>',
+    'support_reached_email_body'         => 'Hallo,<br><br>Gute Nachrichten! Dein Antrag „%TITLE%“ hat die Mindestzahl an Unterstützer*innen erreicht.<br><br>Jetzt bist du noch einmal am Zug. Bitte reiche deinen Antrag nun offiziell bei der Antragskommission ein. Dies kannst du über das SPD-Antragstool unter dem folgenden Link machen:<br><br><strong>%LINK%</strong><br><br>Der Antrag wird zeitnah nach einer formalen Prüfung im SPD-Antragstool veröffentlicht.<br><br>Solidarische Grüße,<br>  SPD Thüringen',
+
+    'proposal_email_title'    => 'Antrag %PREFIX%: Verfahrensvorschlag',
+    'proposal_email_modified' => "Hallo %NAME_GIVEN%,\n\n" .
+                                 "die Programmkommission hat über deinen Antrag beraten und will vorschlagen, ihn modifiziert zu übernehmen.\n\n" .
+                                 "Bitte überprüfe unter folgendem Link, ob du mit der modifizierten Übernahme so einverstanden bist:\n%LINK%\n\n" .
+                                 "Solidarische Grüße,\n" .
+                                 "  SPD Thüringen",
+    'proposal_email_accepted' => "Hallo %NAME_GIVEN%,\n\n" .
+                                 "zu Deiner Information: die Programmkommission wird vorschlagen, deinen Antrag zu übernehmen.\n\n" .
+                                 "Du kannst ihn hier einsehen: %LINK%\n\n" .
+                                 "Solidarische Grüße,\n" .
+                                 "  SPD Thüringen",
+    'proposal_email_other'    => "Hallo %NAME_GIVEN%,\n\n" .
+                                 "die Programmkommission einen Verfahrensvorschlag für deinen Antrag erarbeitet.\n\n" .
+                                 "Bitte überprüfe unter folgendem Link, ob du damit einverstanden bist:\n%LINK%\n\n" .
+                                 "Hinweis: wenn du den eben genannten Link weiterleitest, kann die empfangende Person den Vorschlag damit ebenfalls einsehen und ggf. bestätigen.\n\n" .
+                                 "Solidarische Grüße,\n" .
+                                 "  SPD Thüringen",
+    'resolution_on'           => 'dort beschlossen am',
```

Comment:
kleiner Hinweis am Rande: das ist vllt auch ne Änderung die sich im de Paket lohnt. Das "dort" ist neu, und macht noch mal klar wo es beschlossen wurde (in der Unterstruktur) - das hat bei uns teilweise Verwirrung ausgelöst, instbesondere wenn nicht alle Anträge dort einen Eintrag hatten.